eq-author-api/schema/tests/questionnaire.test.js

```diff
@@ -53,11 +53,6 @@ describe("questionnaire", () => {
         )
       );
     });
-
```

Comment:
I think this should still be a test.  Can we test another way by calling the mutation directly? eg `createSection(...)`